admin-dev/themes/new-theme/js/components/country-state-selection-toggler.js

```diff
@@ -52,47 +52,52 @@ export default class CountryStateSelectionToggler {
     this.$countryStateSelector = $(countryStateSelector);
     this.$countryInput = $(countryInputSelector);
 
-    this.$countryInput.on('change', () => this._toggle());
+    this.$countryInput.on('change', () => this._change());
 
     // toggle on page load
-    this._toggle(true);
+    this._toggle();
 
     return {};
   }
 
   /**
-   * Toggles State selection
+   * Change State selection
    *
    * @private
    */
-  _toggle(isFirstToggle = false) {
+  _change() {
+    const countryId = this.$countryInput.val();
+    if (countryId === '') {
+      return;
+    }
     $.ajax({
```

Comment:
```suggestion
    $.get({
```
And remove `method: 'GET'`